schematic/store/synapse.py

```diff
@@ -363,47 +363,67 @@ def updateDatasetManifestFiles(self, datasetId: str) -> str:
 
         return manifest_id
 
-    def getAllManifests(self) -> List[str]:
-        """Gets all metadata manifest files across all datasets in projects a user has access to.
 
-        Returns: A list of projects, datasets per project and metadata manifest Synapse ID for each dataset
+    def getAllManifests(self, projectId: str) -> List[str]:
```

Comment:
Change method name to something that corresponds to the new functionality, i.e., get all manifests specific to a project. Consider naming it to `getProjectManifests()`.

Reply:
addressed in commit.